config/webpack.config.js

```diff
@@ -3,41 +3,44 @@
 
 var path = require('path');
 var webpack = require('webpack');
-var ForemanVendorPlugin = require('@theforeman/vendor').WebpackForemanVendorPlugin;
+var ForemanVendorPlugin = require('@theforeman/vendor')
```

Comment:
Formatting changes in the same commit as actual changes is very painful for a reviewer. Please split it out into two commits.